modules/contact-form/js/grunion-frontend.js

```diff
@@ -1,6 +1,3 @@
 jQuery( function ( $ ) {
-	var datefield = $( '.contact-form input[type="date"]' );
-	if ( 'function' === typeof $.fn.datepicker && 'text' === datefield[0].type ) {
-		datefield.datepicker( { dateFormat : 'yyyy-mm-dd' } );
-	}
+	$( '.contact-form input.date' ).datepicker( { dateFormat : 'yy-mm-dd' } );
```

Comment:
Should we find a way to scope this better?  These are pretty generic classes and it's technically possible (yet unlikely) that there will be another form with the same classes on the same page in a sidebar/widget whatever.

Reply:
Updated the class to `jp-contact-form-date`.
The input class is pulled from the shortcode, so I added a check in `render()` to override the default for date type fields.
 
This could be resolved more elegantly in the future via #5876